agent/vendor/github.com/aws/amazon-ecs-agent/ecs-agent/api/ecs/client/ecs_client.go

```diff
@@ -21,6 +21,13 @@ import (
 	"strings"
 	"time"
 
```

Comment:
I'm assuming these files are copied over automatically into the vendor directory because this is where other packages will consume it from?